libsql/src/local/database.rs

```diff
@@ -320,7 +324,10 @@ impl Database {
 
     #[cfg(feature = "replication")]
     /// Sync with primary at least to a given replication index
-    pub async fn sync_until(&self, replication_index: FrameNo) -> Result<crate::database::Replicated> {
+    pub async fn sync_until(
+        &self,
+        replication_index: FrameNo,
+    ) -> Result<crate::database::Replicated> {
```

Comment:
Please avoid reformattings like that in PRs that are supposed to do something non-trivial.

Reply:
Seems like rustfmt wasn't run on this file (I have it run on save), what we should have is CI enforcing that we don't merge PRs without it passing rustfmt checks so I will add that.

Reply:
Really odd, my editor is auto formatting this but with `cargo fmt` which we use to check on CI so we avoid these changes its not formatting lines with `#[cfg(feature = "")]`. Gonna take a look into this, seems really odd that cargo fmt isn't picking this up.

Reply:
I wouldn't invest time into that. Just when you see stuff like this put it on a separate PR and even merge it without review.